modules/core/src/main/scala/lucuma/odb/api/model/ConstraintSetModel.scala

```diff
@@ -31,18 +28,31 @@ object ConstraintSetModel extends ConstraintSetModelOptics {
 
   implicit val EqConstraintSet: Eq[ConstraintSetModel] = Eq.fromUniversalEquals
 
-  val AnyConstraints: ConstraintSetModel =
+  /**
+   * The loosest possible observing constraints.
+   */
+  val Any: ConstraintSetModel =
     ConstraintSetModel(
-      NonEmptyString.unsafeFrom("Anything Goes"),
       ImageQuality.TwoPointZero,
       CloudExtinction.ThreePointZero,
       SkyBackground.Bright,
       WaterVapor.Wet,
       ElevationRangeModel.airmassRange.reverseGet(AirmassRange.AnyAirmass)
     )
 
+  /**
+   * Constraints that are set by default when a new observation is created.
+   */
+  val Default: ConstraintSetModel =
+    ConstraintSetModel(
+      ImageQuality.PointEight,
+      CloudExtinction.PointThree,
+      SkyBackground.Bright,
+      WaterVapor.Wet,
+      ElevationRangeModel.airmassRange.reverseGet(AirmassRange.AnyAirmass)
```

Comment:
It looks like this is Airmass 1.0 - **3.0**. I think Andy wants 1.0 - **2.0**

Reply:
Thanks, that makes more sense for a default.  I'll put in a new PR for it.